src/Draco.Compiler/Internal/Codegen/MetadataCodegen.cs

```diff
@@ -224,17 +222,7 @@ private void EncodeAssembly()
            @namespace: "System",
            name: "Object");
 
-        // Create the free-functions type
-        this.AddTypeDefinition(
-            attributes: TypeAttributes.Public | TypeAttributes.Class | TypeAttributes.AutoLayout | TypeAttributes.BeforeFieldInit | TypeAttributes.Abstract | TypeAttributes.Sealed,
-            @namespace: default,
-            name: "FreeFunctions",
-            baseType: systemObject,
-            // TODO: Again, this should be read up from an index
-            fieldList: MetadataTokens.FieldDefinitionHandle(1),
-            // TODO: This depends on the order of types
-            // we likely want to read this up from an index
-            methodList: MetadataTokens.MethodDefinitionHandle(1));
+        this.EncodeModule((OptimizingIr.Model.Module)this.assembly.RootModule, systemRuntime, systemObject);
```

Comment:
Why do we need a Module here? We aren't in the optimizer anymore, we should take the readonly interfaces